### ISSUE TITLE
WSA Not Able to be installed on Win 11 22H2 ARM64

### ISSUE THREAD

**Issue by coderUT** [microsoft/WSA#212]
### Steps to reproduce

Clean Windows 11 Install w/Virtual Machine Platform enabled
Open Microsoft Store
Search for "Amazon Appstore"

### ✔️ Expected Behavior

I am submitting this issue here because submitting it in the Windows forums gets either no response OR try DISM rebase (which I have already tried).

Windows 11 Enterprise 22H2, ARM64
Surface Pro X (SQ1) w/16GB RAM
Virtual Machine Platform and Hyper-V enabled (I can disable Hyper-V and just leave Virtual Machine Platform and it still dosen't work)
Microsoft Store: 22204.1400.4.0
Local: United States

I should be able to install "Amazon App Store" from Microsoft Store (which should also install WSA)  

### ❌ Actual Behavior

I get an error message that my computer lacks the minimum required memory, even though it has 16GB.
![WindowsStoreError](https://user-images.githubusercontent.com/30981676/214102526-cd7a6c8b-0d37-4731-b552-871ec3feb7a3.png)
![SystemInfo](https://user-images.githubusercontent.com/30981676/214102639-96a92072-e5de-4c3f-82f4-44cbfb3f41a8.png)

### Please specify the version of Windows Subsystem for Android

I can't install it, so I don't know

**Comment by jaholme** [comment]
For the category select Microsoft Store > "Downloading, installing and updating"

**Comment by coderUT** [comment]
Ok, I will so that. Thanks for your response.

**Comment by coderUT** [comment]
That should have said "I will do that". Also, I don't want to close this until there is some resolution.

**Comment by dpaulino** [comment]
@coderUT after you've created the feedback hub report, can you share its link here? I'm an engineer on the store app team and I can take a look.

**Comment by coderUT** [comment]
I'm going to have to go ahead and close this, thanks for the response though. 

It turns out that in our domain, group policies do not allow any Internet connections to Windows Update. As a result, the Microsoft Store app cannot install any apps (I found this out when I tried to install the Feedback Hub app).

That being said, I still think there is an issue with hardware detection; however, even if it was fixed, I couldn't install the Amazon App Store and WSA due to my domain's group policy settings.
